config/github_config.exs

```diff
@@ -5,5 +5,11 @@ github_access_key = case Mix.env do
   otherwise -> System.get_env "GITHUB_ACCESS_KEY"
 end
 
+org_name = case Mix.env do
```

Comment:
I think it's more idomatic to have another file for test, etc. Doesn't really matter, but maybe we'll change this later.